inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -654,7 +654,28 @@ class DataQuantizer<Desc, InferenceEngine::WeightableLayer *> : public DataQuant
  public:
     explicit DataQuantizer(float scaleFactor) : DataQuantizerBase(scaleFactor) {}
     bool operator()(InferenceEngine::WeightableLayer *wl) const {
-        quantizeWeightsBiases<typename Desc::MandatoryType>(Desc::mandatory(), wl, Quant<typename Desc::MandatoryType>());
+        auto quantData = InferenceEngine::getInjectedData<QuantizedLayerParams>(*wl);
+        if (quantData->_weights_quant.GetLevels() <= 255) {
```

Comment:
it's better to use std::numeric_limits<uint8_t> instead of a magic number

Reply:
done